### DISCUSSION TITLE
Problemas na Autenticação do sulla!

### DISCUSSION THREAD

**Discussion by vinnyCalixto** [HelioSilva/Whatsapp-bot-NodeJS#8] [Q&A]
Boa tarde Helio, no processo de autenticação, mesmo lendo o QR Code ele não autentica! Já tem informações sobre ou solução?

Obrigado

**Comment by HelioSilva** [comment]
Boa tarde Vinny, 

Já vi que encontrou a solução usando o Venom-bot, por sinal o repositório já está usando o Venom-bot.

**Reply by vinnyCalixto** [reply]
Fala Helio, muito obrigado! Consegui sim, ia te perguntar se é possível redirecionar para outro whatsapp, após a seleção de uma área da empresa? Obrigado desde já....
